.changeset/olive-eggs-accept.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/backend-common': minor
+---
+
+**BREAKING**: Server-to-server authentication tokens issued from a
```

Comment:
Interesting question, up for debate, is this actually a breaking change?

Reply:
Yeah I think strictly it is, it really leads to a significant change in externally visible behavior - but then again, you were never meant to use tokens in a way that could lead to expiration. So it straddles between breaking and just bug fix if a behavior that should have always been there

Reply:
changed to warnings, nerfed to deprecation